polylogue/automation.py

```
@@ -0,0 +1,183 @@
+TARGETS = _load_targets()
+
+
+def resolve_target(target: str) -> AutomationTarget:
+    try:
+        return TARGETS[target]
+    except KeyError as exc:  # pragma: no cover - defensive
+        raise ValueError(f"Unsupported automation target: {target}") from exc
+
+
+def build_command(target: AutomationTarget, extra_args: Iterable[str]) -> str:
+    parts = [sys.executable, str(SCRIPT_PATH)] + target.command + list(extra_args)
```

Ruff (0.15.9)

[warning] Consider iterable unpacking instead of concatenation

Replace with iterable unpacking

(RUF005)